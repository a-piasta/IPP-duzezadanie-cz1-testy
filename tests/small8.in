newRoute;558;ijmlvriaawbxhd;ijmlvriaawbxhd
repairRoad;ijmlvriaawbxhd;lnivkktddaxzerrp;-14
newRoute;785;yhucmbubwhafivrvfmhphkuii;fbymckiovionvamnkvcvstfs
repairRoad;ijmlvriaawbxhd;pliawhwdmdwdjmzv;0
newRoute;272;yhucmbubwhafivrvfmhphkuii;pliawhwdmdwdjmzv
repairRoad;fbymckiovionvamnkvcvstfs;bjjxwhcerrepwmxarmsvhcaolwcm;-1
repairRoad;fbymckiovionvamnkvcvstfs;bjjxwhcerrepwmxarmsvhcaolwcm;14
newRoute;530;fbymckiovionvamnkvcvstfs;mbnbuniqrxrlantbvbwmybaf
newRoute;769;fbymckiovionvamnkvcvstfs;pliawhwdmdwdjmzv
repairRoad;bjjxwhcerrepwmxarmsvhcaolwcm;pliawhwdmdwdjmzv;8
extendRoute;785;mbnbuniqrxrlantbvbwmybaf
extendRoute;272;coywkfamvmjyypnlz
newRoute;966;yhucmbubwhafivrvfmhphkuii;durlzfqbdffqptlxjy
extendRoute;530;mbnbuniqrxrlantbvbwmybaf
newRoute;587;bjjxwhcerrepwmxarmsvhcaolwcm;durlzfqbdffqptlxjy
getRouteDescription;769
getRouteDescription;272
getRouteDescription;966
addRoad;pliawhwdmdwdjmzv;ijmlvriaawbxhd;10;9
addRoad;mbnbuniqrxrlantbvbwmybaf;fbymckiovionvamnkvcvstfs;25;0
newRoute;942;coywkfamvmjyypnlz;yhucmbubwhafivrvfmhphkuii
repairRoad;yhucmbubwhafivrvfmhphkuii;fnpcukffhumzkn;-2
getRouteDescription;530
newRoute;431;fbymckiovionvamnkvcvstfs;lnivkktddaxzerrp
extendRoute;942;bjjxwhcerrepwmxarmsvhcaolwcm
getRouteDescription;410
extendRoute;942;ijmlvriaawbxhd
removeRoad;fbymckiovionvamnkvcvstfs;mbnbuniqrxrlantbvbwmybaf
getRouteDescription;530
getRouteDescription;769
removeRoad;fbymckiovionvamnkvcvstfs;mbnbuniqrxrlantbvbwmybaf
extendRoute;769;lnivkktddaxzerrp
removeRoad;mbnbuniqrxrlantbvbwmybaf;fbymckiovionvamnkvcvstfs
extendRoute;558;mbnbuniqrxrlantbvbwmybaf
removeRoad;mbnbuniqrxrlantbvbwmybaf;fbymckiovionvamnkvcvstfs
newRoute;586;durlzfqbdffqptlxjy;coywkfamvmjyypnlz
extendRoute;769;mbnbuniqrxrlantbvbwmybaf
getRouteDescription;431
newRoute;934;coywkfamvmjyypnlz;fbymckiovionvamnkvcvstfs
repairRoad;coywkfamvmjyypnlz;mbnbuniqrxrlantbvbwmybaf;8
newRoute;217;durlzfqbdffqptlxjy;bjjxwhcerrepwmxarmsvhcaolwcm
newRoute;622;yhucmbubwhafivrvfmhphkuii;bjjxwhcerrepwmxarmsvhcaolwcm
removeRoad;ijmlvriaawbxhd;pliawhwdmdwdjmzv
getRouteDescription;217
newRoute;16;fnpcukffhumzkn;yhucmbubwhafivrvfmhphkuii
getRouteDescription;934
removeRoad;pliawhwdmdwdjmzv;ijmlvriaawbxhd
repairRoad;yhucmbubwhafivrvfmhphkuii;ijmlvriaawbxhd;-1
getRouteDescription;622
repairRoad;ijmlvriaawbxhd;mbnbuniqrxrlantbvbwmybaf;-13
repairRoad;lnivkktddaxzerrp;lnivkktddaxzerrp;9
repairRoad;coywkfamvmjyypnlz;pliawhwdmdwdjmzv;-2
extendRoute;586;coywkfamvmjyypnlz
removeRoad;mbnbuniqrxrlantbvbwmybaf;fbymckiovionvamnkvcvstfs
