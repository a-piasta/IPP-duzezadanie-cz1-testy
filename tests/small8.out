1: NIE
2: NIE
3: NIE
4: NIE
5: NIE
6: NIE
7: NIE
8: NIE
9: NIE
10: NIE
11: NIE
12: NIE
13: NIE
14: NIE
15: NIE
16: 
17: 
18: 
19: TAK
20: NIE
21: NIE
22: NIE
23: 
24: NIE
25: NIE
26: 
27: NIE
28: NIE
29: 
30: 
31: NIE
32: NIE
33: NIE
34: NIE
35: NIE
36: NIE
37: NIE
38: 
39: NIE
40: NIE
41: NIE
42: NIE
43: TAK
44: 
45: NIE
46: 
47: NIE
48: NIE
49: 
50: NIE
51: NIE
52: NIE
53: NIE
54: NIE
